tests/integrationv2/test_serialization.py

```diff
@@ -17,6 +17,11 @@
 SERVER_DATA = f"Some random data from the server:" + random_str(10)
 CLIENT_DATA = f"Some random data from the client:" + random_str(10)
 
+S2N_TEST_POLICIES = {
+    Protocols.TLS12.value: Ciphers.SECURITY_POLICY_DEFAULT,
```

Comment:
If we update the default policy to support TLS 1.3, I think we'd silently lose TLS 1.2 coverage here. Should the TLS 1.2 policy maybe be pinned? Or maybe assert that TLS 1.2 is negotiated in these tests?

Reply:
Hmmm yeah maybe using the security policy 20210816 is better here? That one doesn't have TLS1.3 cipher suites.